typescript/web/heap.ts

```diff
@@ -0,0 +1,19 @@
+let customDestination = {
+  logEvent: function (eventName, eventProperties) {
+    window.heap.track(
+      eventName,
+      eventProperties as Record<string, unknown>
+    )
+  },
+  setUserProperties: function (userId, userProperties) {
+    window.heap.addUserProperties(
+      userProperties as Record<string, unknown>
+    )
+  },
+  identify: function (userId) {
+    window.heap.identify(userId)
+  },
+  unidentify: function () {
+    window.heap.resetIdentity()
+  },
+}
```

Comment:
Should we include all methods in the example, to prevent people from accidentally forgetting to handle some that they actually need?